config/build/webpack.config.common.js

```diff
@@ -82,7 +85,12 @@ export function config(options) {
             new LoaderOptionsPlugin({
                 options: {
                     context: __dirname,
-                    postcss: [autoprefixer()],
+                    postcss: [
+                        postcssImport(),
```

Comment:
as specified by project readme

> This plugin can consume local files, node modules or web_modules. To resolve path of an @import rule, it can look into root directory (by default process.cwd()), web_modules, node_modules or local modules. When importing a module, it will look for index.css or file referenced in package.json in the style or main fields.